dll/win32/shell32/lang/da-DK.rc

```diff
@@ -1009,7 +1009,7 @@ BEGIN
     IDS_COPYITEMS "Copy Items"
     IDS_COPYBUTTON "Copy"
     IDS_MOVETOMENU "Move to &folder..."
-    IDS_MOVETOTITLE "Select the place where you want to move '%s' to. Then click the Move button."
+    IDS_MOVETOTITLE "Wählen Sie den Ort, in den '%s' verschoben werden soll. Klicken Sie anschließend auf Verschieben."
```

Comment:
Sorry about this one. This is still not correct. My fault. I did not intend to propose a change to da-DK. It was an off-by-one from my side. Better just leave the english text here for now.

Reply:
OK.